health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -151,12 +151,18 @@ const Hypothesis = ({ category, assumptions: initialAssumptions, setShowToast, a
         {assumptions.map((item, index) => {
+          const sourceCheck = assumptionValues?.find((i) => i?.key === item)?.source;
           return (
             <LabelFieldPair className="mp-hypothesis-label-field" style={{ alignItems: "center"}} key={index}>
               <div className="assumption-label">
                 <span>
                   {`${t(item)}`}
-                  <span className="mandatory-span">*</span>
+                  {/* {category === "CAMPAIGN_VEHICLES" || sourceCheck === "CUSTOM" ? null : <span className="mandatory-span">*</span>} */}
+                  {category === "CAMPAIGN_VEHICLES" || sourceCheck === "CUSTOM" ? null : (
+                    <span className="icon-wrapper">
+                      <TooltipWrapper content={t(`HYPOTHEISIS_MESSAGE_FOR_${item}`)} children={<InfoBannerIcon fill={"#C84C0E"} />} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)